cadcdata/cadcdata/storageinv.py

```diff
@@ -382,6 +413,8 @@ def cadcget(self, id, dest=None, process_bytes=None):
                         urls.count(url) < MAX_TRANSIENT_TRIES:
                     # this is a transient exception - append url to try later
                     urls.append(url)
+                if urls:
+                    logger.debug('Try the next URL')
```

Comment:
I would put this comment in the scope of the `if` statement at line 412.

There is already the logging message at the start of the `for` loop at l357,  so every iteration through the loop is already logged, and when trying to figure out what's changing, _I_ think it would be good to know when a URL is added to the list.


This comment is my opinion only, and you may ignore it.

Reply:
That message is there to show why the code goes back to l357 in case it doesn't exit the loop when it should (after a successful get). But I agree that the `TransientException` case needs a debug message which I've added.